datalab/stackdriver/monitoring/_resource.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied.  See the License for the specific language governing permissions and limitations under
+# the License.
+
+"""Provides the ResourceDescriptors in the monitoring API."""
+
+from __future__ import absolute_import
+from builtins import object
+
+import collections
```

Comment:
Thanks @parthea. I have addressed the `flake8` errors other than:

* `E111` indentation is not a multiple of four
* `F401` imported but unused in `__init__.py`
* `E402` module level import not at top of filesome local imports in `test/main.py`
* `E501` line too long - mostly in the copyright